### PR TITLE
 Changed project name `pybamm-cookiecutter` -> `pybamm-cookie`

### PR DESCRIPTION
**Pull request by santacodes**
Changed all instances of `pybamm-cookiecutter` to `pybamm-cookie`.

#26

### CODE REVIEW

**Review by Saransh-cpp — APPROVED** [review]
Thanks, @santacodes! Looks good to me!

File: src/pybamm_cookie/__init__.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+"""
+Copyright (c) 2023 The PyBaMM Team. All rights reserved.
```
Comment (agriyakhetarpal) [inline src/pybamm_cookie/__init__.py:2]:
```suggestion
Copyright (c) 2024 The PyBaMM Team. All rights reserved.
```

File: .github/workflows/test_on_push.yml [hunk 2]
```diff
@@ -6,6 +6,7 @@ on:
   push:
     branches:
     - main
+    - pybammcookie
```
Comment (agriyakhetarpal) [inline .github/workflows/test_on_push.yml:9]: I'm not sure why this is required?

**Review by agriyakhetarpal — APPROVED** [review]
Thanks, @santacodes!

File: .github/workflows/test_on_push.yml (hunk 2 above)
Comment (santacodes) [reply .github/workflows/test_on_push.yml:9]: Oops I was testing it locally forgot to remove it 😅

**Pull request merged**